src/Humanizer/Humanizer.csproj

```diff
@@ -244,9 +245,6 @@
   <ItemGroup>
     <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetPortable)' == 'false' ">
```

Comment:
Hm good question, it's present on my local system but not in my branched repository. And I don't have any changes to commit.